yt_dlp/extractor/vidio.py

```diff
@@ -56,9 +60,28 @@ def _real_extract(self, url):
             })
         video = data['videos'][0]
         title = video['title'].strip()
+        hls_url = data['clips'][0]['hls_url']
+
+        formats, subs = self._extract_m3u8_formats_and_subtitles(
+            hls_url, display_id, 'mp4', 'm3u8_native', errnote=False, fatal=False)
+        if not formats:
+            self.to_screen('Falling back to premier mode.')
+            sources = self._download_json(
+                'https://www.vidio.com/interactions_stream.json?video_id=' + video_id + '&type=videos', display_id)
+            if not (sources.get('source') or sources.get('source_dash')):
+                self.raise_login_required()
+
+            if sources.get('source'):
+                hls_formats, hls_subs = self._extract_m3u8_formats_and_subtitles(
+                    sources['source'], display_id, 'mp4', 'm3u8_native')
+                formats.extend(hls_formats)
+                subs.update(hls_subs)
+            if sources.get('source_dash'):  # TODO: Find video example with source_dash, can both exist at the same time?
```

Comment:
```suggestion
            if sources.get('source_dash'):  # TODO: Find video example with source_dash
```
> can both exist at the same time?

The code can already handle either case, so no need to put that in todo

Reply:
done